shared/components/BurgerMenu/BurgerMenuLocalesList.vue

```diff
@@ -16,14 +16,63 @@
       {{ localeProperties.name }}
     </button>
     <ul ref="localeSwitcher" class="burger-menu-locales-list__items" tabindex="0">
-      <li v-for="locale in locales" :key="locale.code">
+      <li v-if="frLocale">
```

Comment:
Plutôt boucler sur `runtimeConfig.public.availableLocales` nn ?
Ca éviterait de répeter le template X fois par locale disponible 😄 

On peut si besoin définir le script une constante : 
```
const availableLocales = runtimeConfig.public.availableLocales
```

et ici : 
```suggestion
      <li v-for="locale in availableLocales" :key="locale.code">
```